### ISSUE TITLE
Installed it & applied, lost all search windows (local) capability

### ISSUE THREAD

**Issue by MonykaR** [10se1ucgo/DisableWinTracking#109]
**INFORMATION:**

* Windows Edition (Home/Pro/Enterprise): 
* Architecture (32bit/64bit): 
* Options used:

**LOG:**

```python
REPLACE THIS WITH DISABLEWINTRACKING.LOG
```

**Comment by brainscript** [comment]
@MonykaR Are you thinking about filling in the info fields?!

**Comment by MonykaR** [comment]
Well, I lost the log when I closed the output window and now of course if I re-run the disable or delete and the uninstall, it is empty. I can revert and re-install if you like.

**Comment by 10se1ucgo** [comment]
The program creates a file named 'DisableWinTracking.log' in the directory you ran it from.

**Comment by MonykaR** [comment]
I looked there as well, it's completely empty... 

I re-ran the uninstall - all it gave was : 
13:40:28 INFO: DisableWinTracking Version: v2.5.1

I can't undo the delete of the privacy section, however :( so I cannot re-create the log, but I looked it at yesterday and it didn't say very much.

**Comment by MonykaR** [comment]
I re-ran the delete:
Hope this helps.

13:42:42 INFO: Performing clutter control

13:42:43 INFO: DiagTrack box ticked

13:42:43 INFO: Service delete option ticked

13:42:43 INFO: DisableWinTracking Version: v2.5.1

13:42:43 INFO: Mode: Privacy

13:42:43 INFO: Telemetry box ticked

13:42:43 INFO: Host box ticked

13:42:43 INFO: Extra host box ticked

13:42:43 INFO: IP block box ticked

13:42:44 INFO: Defender/Wifisense box ticked

13:42:44 ERROR: Registry: Unable to modify Windows Defender Spynet key.
Traceback (most recent call last):
  File "<string>", line 575, in modifyregistry
WindowsError: [Error 5] Access is denied

13:42:44 ERROR: Registry: Unable to modify Windows Defender Sample Submission key.
Traceback (most recent call last):
  File "<string>", line 575, in modifyregistry
WindowsError: [Error 5] Access is denied

13:42:44 INFO: OneDrive box ticked

**Comment by 10se1ucgo** [comment]
I'm having a similar issue, but I've had the issue since before I even started working on this. A restart would usually fix it. Are you sure it's related to DWT?

If the issue bothers you, you can try Everything search, which is miles better than the normal Windows search.

**Comment by MonykaR** [comment]
The search was working perfectly fine until I launched and ran DWT.
I am super annoyed that the search no longer works. I was really enjoyed it as opposed to searching for an app or setting in a menu.

**Comment by Ruined1** [comment]
@MonykaR I'm not saying that it's ok this happened, but at the same time, you should always create a system restore point before making major changes like this in case you do not like the results. Did you try reverting with DWT?

**Comment by 10se1ucgo** [comment]
Still not having this issue

![search](http://i.imgur.com/sv3ugQb.png)

**Comment by MonykaR** [comment]
@Ruined1 Good idea, I will reinstall windows. I guess I was being lazy and really hoping for an easy fix.

**Comment by Loyalar** [comment]
I had the exact same issue as the OP - the search menu was entirely blacked out (an example can be seen at https://gyazo.com/9b27dc09c4559d1dfcb444a4481911e5). I tried re-enabling all the tracking that I had previously disabled, restarted, and then disabled them one at a time, with restarts in between each option. When I checked off "Block tracking IP addresses" and restarted, the issue was there again - and the issue disappeared when I reverted the "Block tracking IP addresses" option. I'm running a 64 bit Windows 10 Pro.

**Comment by MonykaR** [comment]
There was a windows update and search is back!! I may run the trouble shoot approach that @Loyalar Loyalar mentioned, though :)

**Comment by Baigle** [comment]
When switching to Win10 that was my biggest problem with it... you have to kill SearchUI.exe and when it starts back up sometimes it will correctly search and display results in the start menu. It can be influenced negatively by some running "security" software (e.g. Trusteer Rapport, Windows Defender, BitDefender, others)...

you could automate the termination of SearchUI.exe by tying a keyboard shortcut to a script or any other way to taskkill /f /IM SearchUI.exe in windows cmdline

**Comment by MonykaR** [comment]
I did a clean install of Windows 10, reran DWT and made sure nothing interfered with during the uninstall processes, rebooted and everything works fine. Search works! :) Unfortunately, I am really unsure of what went wrong the first time; either I accidentally interrupted the process or the perhaps the disablings that I did manually prior to knowing about DWT had some negative interactive effects with DWT.
